include/graphtyper/graph/log_table.hpp

```diff
@@ -3,1007 +3,207 @@
-  2228.514214268629985,
-  2231.462137888461712,
-  2234.410550854240228,
-  2237.359452615210103,
-  2240.308842621855547,
-  2243.258720325892227,
-  2246.209085180268630,
-  2249.159936639156967,
-  2252.111274157952721,
-  2255.063097193268732,
-  2258.015405202931106,
-  2260.968197645975124,
-  2263.921473982642510,
-  2266.875233674375522,
-  2269.829476183814677,
-  2272.784200974793748,
-  2275.739407512336129,
-  2278.695095262649374,
-  2281.651263693124747,
-  2284.607912272329941,
-  2287.565040470006807,
-  2290.522647757066807,
-  2293.480733605588284,
-  2296.439297488810098,
-  2299.398338881131167,
-  2302.357857258104104,
-  2305.317852096432489,
-  2308.278322873966772,
-  2311.239269069700185,
-  2314.200690163767376,
-  2317.162585637434859,
-  2320.124954973105105,
-  2323.087797654306087,
-  2326.051113165692186,
-  2329.014900993037372,
-  2331.979160623234293,
-  2334.943891544288363,
-  2337.909093245314125,
-  2340.874765216534342,
-  2343.840906949273176,
-  2346.807517935955275,
-  2349.774597670099411,
-  2352.742145646318932,
-  2355.710161360312213,
-  2358.678644308866296,
-  2361.647593989847337,
-  2364.617009902201517,
-  2367.586891545947765,
-  2370.557238422177761,
-  2373.528050033050931,
-  2376.499325881788991,
-  2379.471065472676401,
-  2382.443268311055363,
-  2385.415933903321729,
-  2388.389061756921365,
-  2391.362651380348780,
-  2394.336702283141676,
-  2397.311213975879127,
-  2400.286185970177030,
-  2403.261617778686286,
-  2406.237508915087801,
-  2409.213858894091118,
-  2412.190667231428961,
-  2415.167933443856782,
-  2418.145657049145484,
-  2421.123837566082784,
-  2424.102474514467303,
-  2427.081567415105383,
-  2430.061115789809719,
-  2433.041119161393453,
-  2436.021577053669716,
-  2439.002488991446626,
-  2441.983854500525013,
-  2444.965673107695693,
-  2447.947944340734921,
-  2450.930667728403478,
-  2453.913842800441671,
-  2456.897469087566151,
-  2459.881546121469000,
-  2462.866073434812733,
-  2465.851050561228021,
-  2468.836477035310963,
-  2471.822352392619450,
-  2474.808676169669980,
-  2477.795447903936292,
-  2480.782667133844825,
-  2483.770333398771072,
-  2486.758446239039131,
-  2489.747005195917609,
-  2492.736009811616441,
-  2495.725459629282796,
-  2498.715354193001986,
-  2501.705693047789737,
-  2504.696475739592643,
-  2507.687701815284981,
-  2510.679370822665078,
-  2513.671482310452120,
-  2516.664035828283886,
-  2519.657030926715379,
-  2522.650467157213370,
-  2525.644344072154581,
-  2528.638661224823863,
-  2531.633418169411470,
-  2534.628614461008510,
-  2537.624249655606036,
-  2540.620323310091408,
-  2543.616834982246019,
-  2546.613784230741203,
-  2549.611170615138690,
-  2552.608993695884237,
-  2555.607253034307632,
-  2558.605948192619508,
-  2561.605078733906794
-}};
+std::array<double, 999> static const LOG10_TABLE = {
+  {0.000000000000000,    0.000000000000000,    0.301029995663981,    0.778151250383644,    1.380211241711606,
```

Comment:
You could do the same thing here, although I actually think the multi-column layout isn't bad.